### PR TITLE
Docs: Change relative path to absolute path in md #DS-1364

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

**Comment by literat** [comment]
Looks good. 👍 Still searching for relative paths?

**Review by literat — APPROVED** [review]
👏

File: README.md [hunk 1]
```diff
@@ -44,13 +44,23 @@ See individual [packages](#packages) to learn how to get started.
...
 
-See [`package.json`](./package.json) for all available tasks.
+See [`package.json`][package.json] for all available tasks.
 
 ## License
 
 See the [LICENSE](LICENSE.md) file for information.
 
 [alma-career]: https://github.com/lmc-eu
+[packages-analytics]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/analytics
```
Comment (dlouhak) [inline README.md:54]: Glad that all links are grouped together at the bottom of the file and they are called via reference/key. 🙌🏻

I was thinking if the rest of the files should have same structure, but that's not explicitly mentioned in the issue description, so... 🤷🏻‍♀️

Comment (dlouhak) [reply README.md:57]: Thank you! <3

File: packages/web-twig/README.md [hunk 2]
```diff
@@ -155,5 +155,11 @@ Check your Symfony console or log to see if you are using any of the deprecated
 
 ![Deprecations in Symfony's console](https://github.com/lmc-eu/spirit-design-system/blob/main/static/deprecations-symfony-console.png?raw=true)
 
+[components-directory]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-twig/src/Resources/components
+[contributing]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/CONTRIBUTING.md
+[example-guide]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/docs/extendComponents.md
+[changelog]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/CHANGELOG.md
```
Comment (dlouhak) [inline packages/web-twig/README.md:161]: 🔤 OCD triggered 😄 
Still don't understand, why the English alphabet doesn't have `ch`, such a pretty character.

File: README.md (hunk 1 above)
Comment (curdaj) [reply README.md:62]: Refactored😉

File: packages/web/README.md [hunk 3]
```diff
@@ -190,12 +190,13 @@ Check your browser console to see if you are using any of the deprecated functio
...
-See the [LICENSE](LICENSE.md) file for information.
+See the [LICENSE][license] file for information.
 
 [configuring-load-path]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#configuring-load-path
-[feature-flags-docs]: ../../docs/contributtion/feature-flags.md
-[tokens-api]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#tokens-api
-[rebranding]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/design-tokens#b-via-load-path
-[examples]: https://spirit-design-system.netlify.app/packages/web/
+[deprecations]: https://github.com/lmc-eu/spirit-design-system/blob/main/static/deprecations-browser-console.png?raw=true
+[examples]: https://spirit-design-system.netlify.app/packages/wehttps://github.com/lmc-eu/spirit-design-system/blob/main/docs/contributtion/feature-flags.md
```
Comment (dlouhak) [inline packages/web/README.md:197]:
```suggestion
[examples]: https://spirit-design-system.netlify.app/packages/web/
[feature-flags-docs]: https://github.com/lmc-eu/spirit-design-system/blob/main/docs/contributtion/feature-flags.md
```

**Review by dlouhak — APPROVED** [review]
One small fix but otherwise LGTM 💪🏻

**Pull request merged**